docs/user-guide/cai.rst

```diff
@@ -62,15 +64,10 @@ This section is a step-by-step tutorial that walks you through how to run a full
 
 7. Run inference.
 
-.. note::
-   Before starting this tutorial, be sure to review the :ref:`introduction <model-aligner-intro>` for tips on setting up your NeMo-Aligner environment.
-
-   If you run into any problems, refer to NeMo's `Known Issues page <https://docs.nvidia.com/nemo-framework/user-guide/latest/knownissues.html>`__. The page enumerates known issues and provides suggested workarounds where appropriate.
-
 .. image:: ../assets/cai_flow.png
 
 Step 1: Download models and datasets
```

Comment:
revise

Step 1: Download the models and datasets

Reply:
https://github.com/NVIDIA/NeMo-Aligner/pull/411/commits/9432333a44ad01d9534ed89263a37000b2aaca11